README.md

```diff
@@ -1,159 +1,59 @@
-</p>
+## Features
 
-<br>
+- **Markdown-based Note-Taking**: Create, edit, and save notes using markdown.
+- **File Organization**: Organize notes into folders and subfolders.
 
-<div align="center">
+## Prerequisites
 
-[![Build Status][github-actions-status]][github-actions-url]
-[![Github Tag][github-tag-image]][github-tag-url]
-[![Discord](https://badgen.net/badge/icon/discord?icon=discord&label)](https://discord.gg/Fjy3vfgy5q)
+Before starting, ensure that you have the following tools installed:
 
-[![OpenCollective](https://opencollective.com/electron-react-boilerplate-594/backers/badge.svg)](#backers)
-[![OpenCollective](https://opencollective.com/electron-react-boilerplate-594/sponsors/badge.svg)](#sponsors)
-[![StackOverflow][stackoverflow-img]][stackoverflow-url]
+1. **Node.js** (v20)
 
-</div>
+   - [Official Download Page](https://nodejs.org/en/download/)
+   - Installation Instructions: [How to Install Node.js](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
 
-## Install
+2. **Git**
+   - [Official Download Page](https://git-scm.com/downloads)
+   - Installation Instructions: [How to Install Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
 
-Clone the repo and install dependencies:
+## Installation Steps
+
+- First, clone the repository and install the necessary dependencies:
 
 ```bash
-git clone --depth 1 --branch main https://github.com/electron-react-boilerplate/electron-react-boilerplate.git your-project-name
-cd your-project-name
+git clone https://github.com/Kicchu02/Draft-Craft.git
+cd Draft-Craft
 npm install
 ```
 
-**Having issues installing? See our [debugging guide](https://github.com/electron-react-boilerplate/electron-react-boilerplate/issues/400)**
-
-## Starting Development
-
-Start the app in the `dev` environment:
+- Run application
 
 ```bash
 npm start
 ```
 
-## Packaging for Production
+### Directory Structure
 
-To package apps for the local platform:
+The source code for the React app is located in the `src/renderer` directory. This directory contains all the necessary components, utilities, styles, and other resources for the application.
+
+Here’s a brief overview of the folder structure:
 
 ```bash
-npm run package
+Draft-Craft/
+├── src/                    # Source code of the application
+│   ├── renderer/           # React app source code (located here)
+│   │   ├── components/     # React components
+│   │   ├── utils/          # Utility functions and helpers
+│   │   ├── styles/         # CSS/SCSS files for styling
+│   │   └── assets/         # Static assets like icons or images
+│   ├── main/               # Electron main process files
+│   └── ...
+├── public/                 # Public static files
+├── package.json            # App configuration and dependencies
+├── webpack.config.js       # Webpack bundler configuration
+└── README.md               # Project documentation (this file)
 ```
```

Comment:
This is really not necessary. We will be constantly updating the project. As we write code, we cannot keep updating this and keep track of it. Please remove it.